netaddr.go

```diff
@@ -305,6 +305,73 @@ func (ip IP) IsMulticast() bool {
 	}
 }
 
+// Prefix applies a CIDR mask of leading bits to IP, producing an IPPrefix
+// of the specified length. If IP is the zero value, a zero-value IPPrefix is
```

Comment:
> If IP is the zero value, ...

It's not exactly clear whether this sentence means _with a nil error_ now. Maybe just add `(and a nil error)`.